compiler_opt/rl/inlining/gin_configs/imitation_learning.gin

```diff
@@ -0,0 +1,27 @@
+import compiler_opt.rl.inlining.env
+import compiler_opt.rl.inlining.imitation_learning_config
+import compiler_opt.rl.imitation_learning.generate_bc_trajectories_lib
+
+
+env.InliningForSizeTask.llvm_size_path=''
+
+generate_bc_trajectories_lib.ModuleWorker.clang_path=''
+generate_bc_trajectories_lib.ModuleWorker.mlgo_task_type=@env.InliningForSizeTask
+generate_bc_trajectories_lib.ModuleWorker.policy_paths=[]
+generate_bc_trajectories_lib.ModuleWorker.exploration_policy_paths=[]
+generate_bc_trajectories_lib.ModuleWorker.explore_on_features=None
+generate_bc_trajectories_lib.ModuleWorker.base_path=''
```

Comment:
(as discussed offline) let's go with the patch as-is right now, and in a follow-up we remove this from gin and pipe it through from its flag.